DayrSuryan/DSEthiop12.xml

```diff
@@ -571,29 +571,28 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            names of saints, explicit in <ref target="#ms_i7">Text VII (“The angels are praising Mary...”)</ref>
                         </seg>
                         <date notBefore="1800" notAfter="1899">19th century</date>
-                        <desc>Crude, poorly trained, irregular hand.
-                           Parts of the manuscript were written probably at different times, hence
+                        <desc>Irregular, nearly crude; by a mediocre scribe.
+                           Parts of the text were written probably at different times, hence
                            variations in the hues of the inks, appearance of the
                            script (e.g., slant and thickness of the strokes) and quality of the execution
                            (diminishing after ca. <locus target="#45"/>).
                           The hand seems to be very similar or identical to
-                          <ref type="mss" target="DSEthiop8" corresp="#h1"/>.
+                          <ref type="mss" target="DSEthiop8" corresp="#h1">
```

Comment:
I think the correct way to refer to a hand in another manuscript would be 
`<ref type="hand" corresp="DSEthiop8#h1">`

Reply:
ok